server_test.go

```diff
@@ -111,41 +122,61 @@ func setupLocalServer(t *testing.T, config Config) Server {
 
 // DDMetricsRequest represents the body of the POST request
 // for sending metrics data to Datadog
+// Eventually we'll want to define this symmetrically.
 type DDMetricsRequest struct {
 	Series []DDMetric
 }
 
-func TestLocalServer(t *testing.T) {
-	var ddmetrics DDMetricsRequest
+// TestLocalServerUnaggregatedMetrics tests the behavior of
+// the veneur client when operating without a global veneur
+// instance (ie, when sending data directly to the remote server)
+func TestLocalServerUnaggregatedMetrics(t *testing.T) {
+	// Since we are asserting inside a separate goroutine, we should
+	// always assert that we get to the end of the goroutine.
+	// In the future, if the key call (in this case, server.Flush)
+	// is made asynchronous, we will know if we are ever encountering
+	// a race condition in which our assertions are simply not executing.
+	// (Failures after a test has finished executing will actually cause
+	// the test suite to fail, as long as it is still running, but this is
+	// a failsafe).
+	var RemoteResponses int
+	defer func() {
+		assert.Equal(t, 1, RemoteResponses, "Server did not complete all responses before test terminated!")
```

Comment:
Sure, we can do that. In this case, it's a synchronous call right now, so I thought it'd be simpler to leave a 'stub' vs. using a buffered channel, but we can keep it consistent.